irohad/network/channel_constants.hpp

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_CHANNEL_CONSTANTS_HPP
+#define IROHA_CHANNEL_CONSTANTS_HPP
+
+namespace iroha::network {
+
+  /// Determines maximum packet size can be sent via grpc
+  static constexpr int k_max_msg_size = 128 * 1024 * 1024;
```

Comment:
Naming a bit inconsistent, CamelCase is used in other places for constants.
```suggestion
  static constexpr int kMaxMessageSize = 128 * 1024 * 1024;
```